R/insertPackage.R

```diff
@@ -36,9 +36,12 @@ insertPackage <- function(file,
                           commit=FALSE) {
 
     if (!file.exists(file)) stop("File ", file, " not found\n", .Call=FALSE)
-    
-    ## TODO: make src/contrib if needed
-    if (!file.exists(repodir)) stop("Directory ", repodir, " not found\n", .Call=FALSE)
+
+    if (!file.exists(repodir)) stop("Repository directory ", repodir, " not found\n",
+                                    "If you want to use a checkout of your github pages,\n",
+                                    "add a subdir (e.g. 'drat') and then set ",
+                                    "'option(dratRepo = \"/path/to/githubpage/drat/\")'\n",
+                                    .Call=FALSE)
```

Comment:
I fear this doesn't quite work for the case we already support:  writing to a 'normal' directory in non-git/Github mode, eg on a local (NFS or Samba) file share.  So we can't just talk about git, because there are other git uses.